osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNote.cs

```diff
@@ -301,7 +301,7 @@ public bool OnPressed(KeyBindingPressEvent<ManiaAction> e)
             // The tail has a lenience applied to it which is factored into the miss window (i.e. the miss judgement will be delayed).
             // But the hold cannot ever be started within the late-lenience window, so we should skip trying to begin the hold during that time.
             // Note: Unlike below, we use the tail's start time to determine the time offset.
-            if (Time.Current > Tail.HitObject.StartTime && !Tail.HitObject.HitWindows.CanBeHit(Time.Current - Tail.HitObject.StartTime))
+            if (Time.Current > Tail.HitObject.StartTime && !Tail.HitObject.HitWindows.CanEverBeHit(Time.Current - Tail.HitObject.StartTime))
```

Comment:
what is happening here? why "can *ever* be hit"? as opposed to what, can "sometimes" be hit? what is the meaning of this?

you talk about opening "follow up PRs", yet i am very uncomfortable with the complexity already incurred by this change as is, and think there's something going deeply wrong if things are headed this way. i do not believe it is necessary complexity.

Reply:
This is just a rename, the behavior of this method does not change with this PR. I found the name `CanBeHit` confusing, as when I first saw it, I thought it would be

`Math.Abs(timeOffset) <= WindowFor(LowestSuccessfulHitResult())` (`timeOffset` contained within the hit window range),

but it's supposed to be

`timeOffset <= WindowFor(LowestSuccessfulHitResult())` (always true for negative `timeOffset`).

The phrase "can ever be hit" is taken from pre-PR XML docs.